lib/phoenix/template/eex_engine.ex

```diff
@@ -10,7 +10,7 @@ defmodule Phoenix.Template.EExEngine do
   end
 
   defp engine_for(name) do
-    case Phoenix.Template.format_encoder(name) do
+    case Phoenix.Template.format_encoder(Mix.Phoenix.otp_app(), name) do
```

Comment:
I thought it was okay since the engine is only compiled time, but would prefer to avoid it